.github/workflows/coverage.yml

```
@@ -0,0 +1,23 @@
+name: Post coverage comment
+
+on:
+  workflow_run:
+    workflows: ["continuous-integration"]
+    types:
+      - completed
+
+jobs:
+  test:
+    name: Run tests & display coverage
+    runs-on: ubuntu-latest
+    if: github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success'
+    permissions:
+      pull-requests: write
+      contents: write
```

```suggestion
      contents: write # needed to edit the comment vs opening multiple ones
```
This surprised me at first but upon checking [the docs for the action](https://github.com/py-cov-action/python-coverage-comment-action) it made sense so let's add a comment